File: javascript/syntax/3.variables.js
let integer = 12;
let str = "Saransh";
let float = 20.5;
let bool = true;
let nullValue = null;
let [a, b, c] = ["Orange", "Banana", "Cherry"]; //multiple declaration
let list = ["apple", "banana", "cherry"];
let tupple = ["apple", "banana", "cherry"]; // JavaScript doesn't have tuples, using array
let range = Array.from({length: 6}, (_, i) => i);
let dictionary = {"name": "John", "age": 36};
let set = new Set(["apple", "banana", "cherry"]);

console.log(integer); //12
console.log(str); //Saransh
console.log(float); //20.5
console.log(nullValue); //null
console.log(a, b, c); //Orange Banana Cherry
console.log(list); //['apple', 'banana', 'cherry']
console.log(tupple); //['apple', 'banana', 'cherry']
console.log(range); //[0, 1, 2, 3, 4, 5]
console.log(dictionary); //{name: 'John', age: 36}
console.log(set); //Set(3) {'banana', 'apple', 'cherry'} 

File: java/syntax/3.Variables.java
// Variables
import java.util.*;

public class Variables {
    public static void main(String[] args) {
        int integer = 12;
        String str = "Saransh";
        double floatVar = 20.5;
        boolean bool = true;
        Object nullValue = null;
        
        // Multiple declaration
        String a = "Orange", b = "Banana", c = "Cherry";
        
        // Arrays
        String[] arr = {"apple", "banana", "cherry"};
        
        // Tuple equivalent (using Object array)
        Object[] tuple = {"apple", "banana", "cherry"};
        
        // HashMap
        HashMap<String, Object> hashMap = new HashMap<>();
        hashMap.put("name", "John");
        hashMap.put("age", 36);
        
        // Set
        HashSet<String> set = new HashSet<>();
        set.add("apple");
        set.add("banana");
        set.add("cherry");
        
        // Infinity
        double infinite = Double.POSITIVE_INFINITY;

        System.out.println(integer); // 12
        System.out.println(str); // Saransh
        System.out.println(floatVar); // 20.5
        System.out.println(nullValue); // null
        System.out.println(a + " " + b + " " + c); // Orange Banana Cherry
        System.out.println(Arrays.toString(arr)); // [apple, banana, cherry]
        System.out.println(Arrays.toString(tuple)); // [apple, banana, cherry]
        System.out.println(hashMap); // {name=John, age=36}
        System.out.println(set); // [banana, apple, cherry]
        System.out.println(infinite); // Infinity
    }
} 